contract.js

```diff
@@ -66,6 +66,7 @@ const initialize = () => {
   }
   let accounts
   let piggybankContract
+  let accountButtonsInitialized = false
```

Comment:
This should be set to `false` again if the user disconnects from the dapp

Reply:
Actually... maybe they don't need to be re-initialized ever? 🤔

Reply:
Yeah - while initializing them multiple times isn't harmful, it doesn't seem necessary either.

I'd go a step further and initialize them right away upon pageload, once, rather than waiting for the first connection and using this boolean to stop it the second time.